docs/topics/dokka-migration.md

```diff
@@ -51,7 +51,7 @@ Ensure that your project meets the minimum version requirements:
 
 2. In the project's `gradle.properties` file, set the following opt-in flag with helpers to activate the new plugin version:
 
-   ```kotlin
+   ```properties
```

Comment:
Last time I tried to add `properties` in kotlinlang.org docs I was told it doesn't work. Maybe that has changed after the Writerside update?

It would be nice to highlight the codeblock with properties.

Reply:
I checked but unfortunately, it's still not supported.
<img width="864" alt="Screenshot 2024-10-10 at 1 32 15 PM" src="https://github.com/user-attachments/assets/036b82ff-1c16-46ff-ac9f-64ba4e6122ce">